jlm/llvm/opt/alias-analyses/Steensgaard.cpp

```diff
@@ -1060,10 +1061,22 @@ Steensgaard::AnalyzeSimpleNode(const jlm::rvsdg::SimpleNode & node)
   {
     AnalyzeVaList(node);
   }
+  else if (is<PointerToFunctionOperation>(&node))
+  {
+    AnalyzePointerToFunction(node);
+  }
+  else if (is<FunctionToPointerOperation>(&node))
+  {
+    AnalyzeFunctionToPointer(node);
+  }
   else if (is<FreeOperation>(&node) || is<ptrcmp_op>(&node))
   {
     // Nothing needs to be done as FreeOperation and ptrcmp_op do not affect points-to sets
   }
+  else if (is<FunctionToPointerOperation>(&node))
+  {
+    AnalyzeFunctionToPointer(node);
+  }
```

Comment:
This is redundant, you already have it inserted above, no?

Reply:
indeed, removed